src/test/resources/data/InstructorSearchPageUiTest.json

```diff
@@ -534,6 +564,28 @@
       "commentText": {
         "value": "Instructor 1 comment to feedback Question 2"
       }
+    },
+    "comment4FromT1C1ToR1Q2S1C1": {
+      "courseId": "searchUI.idOfTypicalCourse1",
+      "feedbackSessionName": "searchUI.First feedback session",
+      "feedbackQuestionId": "2",
+      "giverEmail": "searchUI.instructor2@course1.tmt",
+      "giverSection": "None",
+      "receiverSection": "None",
+      "feedbackResponseId": "2%searchUI.student3InCourse1@gmail.tmt%searchUI.student2InCourse1@gmail.tmt",
+      "showCommentTo": [
+        "INSTRUCTORS"
+      ],
+      "showGiverNameTo": [
+        "INSTRUCTORS"
+      ],
+      "isVisibilityFollowingFeedbackQuestion": false,
+      "createdAt": "2015-02-01 11:59 PM UTC",
+      "commentText": {
+        "value": "Instructor 3 edited comment to feedback Question 2"
+      },
+      "lastEditorEmail": "searchUI.instructor3@course1.tmt",
+      "lastEditedAt": "2012-02-02 11:59 PM UTC"
```

Comment:
Set `lastEditedAt` to be later than `createdAt`. Having a comment edited before it is created is unlikely to happen under normal usage.